data_structures/linked_list/linked_list.py

```diff
@@ -184,3 +184,18 @@ def kthFromEnd(self, k):
             node = node._next
         raise IndexError('requested node outside linked list length')
 
+
+def mergeLists(ll1, ll2):
+    """merges 2 LL's in zipper style"""
+    x = ll1.head
+    node = x
+    if not x:
+        ll1.head = ll2.head
+        return ll2.head
+    y = ll2.head
+    while x and y:
+        x._next, y = y, x._next
+        x = x._next
+    return ll1
```

Comment:
According to the spec you should be returning a reference to the head of the merged linked list, not the list itself.